lib/Page.js

```diff
@@ -117,12 +117,21 @@ function formatSiteNav(renderedSiteNav) {
   listItems.each(function () {
     // Tidy up the style of each list item
     $(this).attr('style', 'margin-top: 10px');
-    // Found nested list
-    if ($(this).children().first().is('ul')) {
-      const nestedList = $(this).children().first();
-      const dropdownTitle = $(this).contents().first().text();
+    // Do not render dropdown menu for list items with <a> tag
+    if ($(this).children('a').length) {
```

Comment:
Would be better if this was more explicit:

```javascript
if ($(this).children('a').length > 0) {
```

Reply:
It follows `Javascript` style where we use the `truthy` values from an integer directly without doing an explicit comparison since it is guaranteed that the value is an integer >= 0.